### DISCUSSION TITLE
애니메이션 변경시 지연

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#551] [5. Q&A]
다음과 같이 총알 발사 버튼을 눌렀을때 클립을 실행 하게 했습니다 
![image](https://user-images.githubusercontent.com/50188795/160558916-bfa8ecf9-9351-4a44-a960-24a360822622.png)
![image](https://user-images.githubusercontent.com/50188795/160558961-42e100b3-3548-4ba9-93ec-ac80ab097430.png)

총 쏘는 애니메이션이 끝나고 다시 달리기를 시작 할 때 애니메이션이 바로 실행 안되고 멈춰 있는 상태에서 
이동을 시작하는것처럼 보입니다 

이것을 해결 할수 있는 방법이 있나요?

https://user-images.githubusercontent.com/50188795/160559731-acfdbbc9-803e-40e7-8acb-dad17a8dda4b.mp4

**Comment by x4cf** [comment]
한 프레임까지 완벽하게 관리하고싶다면 애니메이터를 수정해서 사용하는게 좋을수도 있습니다 :)

**Reply by smilejsu82** [reply]
아하 커스텀 애니메이터를 만들어서 관리 할수도 있군요?
![image](https://user-images.githubusercontent.com/50188795/160569368-17d1a631-d6be-4489-ba8b-a60fcc568fb1.png)

state에 대한 내용은 어디서 찾아 볼수 있나요? 
![image](https://user-images.githubusercontent.com/50188795/160569863-d604498d-d2d7-4fce-ab3f-0c06660638c0.png)

**Reply by x4cf** [reply]
ZepetoAnimator를 드래그앤 드랍으로 에셋에 이동시키면 수정이 가능해요 :)

**Reply by smilejsu82** [reply]
다음과 같이 애니메이션을 리셋 해서 해결 했습니다 
답변 감사합니다 
![image](https://user-images.githubusercontent.com/50188795/160572150-02ab667a-7f4f-47ea-b73c-80c5c1570023.png)


https://user-images.githubusercontent.com/50188795/160572183-6f78dc44-37d5-4bfa-afb3-4696837e25d2.mp4
